realm/realm-library/src/androidTest/java/io/realm/RealmTests.java

```diff
@@ -781,6 +781,7 @@ public void realmConfiguration_fileName() {
     }
 
     @Test
+    @Ignore
```

Comment:
What do you mean `using a custom core`? With this PR, the CI will always run testing with debug core.